taxcalc/records_variables.json

```diff
@@ -4,85 +4,85 @@
       "required": true,
       "type": "int",
       "desc": "Unique positive numeric identifier for filing unit",
-      "form": {"2017": "private info"}
+      "form": {"2017-2017": "private info"}
```

Comment:
@martinholmer 
Could you please explain why you have changed the format for "form" in records_variables.json from 2017 to 2017-2017.